### PR TITLE
Linter setup - Create black.yml

### PR DESCRIPTION
**Pull request by findyoky**

### CODE REVIEW

**Review by wbc2025 — CHANGES_REQUESTED** [review]
This file has a comma instead of a period in the file extension. File should be names black.yml

**Comment by findyoky** [comment]
Corrected the file extension, nice catch!

**Review by findyoky — COMMENTED** [review]
Corrected file extension.

File: .github/workflows/black.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'  # Change to your preferred Python version
```
Comment (elveskevtar) [inline .github/workflows/black.yml:21]: In this case, we are using python 3.12 docker image

**Review by wbc2025 — APPROVED** [review]

**Pull request merged**